frontend/src/bundles/common/components/slider/slider.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Slider } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { useCallback } from 'react';
+
+import { getValidClassNames } from '../../helpers/helpers.js';
+import { defaultExperienceMarks } from './default-values.js';
+import classes from './style.module.css';
+import { defaultSliderStyle } from './styles.js';
+
+type Mark = {
+    label?: string;
+    value: number;
+};
+
+type Properties = {
+    label?: string;
+    value?: number;
+    marks?: Mark[] | boolean | undefined;
```

Comment:
Also you don't need to provide type for undefined like this if you use ? operator
```suggestion
    value?: number;
    marks?: Mark[] | boolean;
```

Reply:
agree, I will remove it